src/ebmc/ebmc_base.cpp

```diff
@@ -910,6 +890,43 @@ int ebmc_baset::get_transition_system()
 
 /*******************************************************************\
 
+Function: ebmc_baset::get_properties
+
+  Inputs:
+
+ Outputs:
+
+ Purpose:
+
+\*******************************************************************/
+
+int ebmc_baset::get_properties()
+{
+  // Property given on command line?
+  if(cmdline.isset('p'))
+  {
+    // NuSMV also uses -p
+    if(parse_property(cmdline.get_value('p')))
+      return 1;
+  }
```

Comment:
Is NuSMV the right role model here? Should we perhaps support both `-p` and `--property` (which is what CBMC uses)?

Reply:
`--property` selects a property, from an identifier, as in CBMC. `-p` takes an expression as argument.